geode-for-redis/src/main/java/org/apache/geode/redis/internal/commands/RedisCommandType.java

```diff
@@ -163,215 +165,295 @@
-  PSUBSCRIBE(new PsubscribeExecutor(), SUPPORTED,
+  PSUBSCRIBE(new PsubscribeExecutor(), Category.PUBSUB, SUPPORTED,
       new Parameter().min(2).firstKey(0).flags(Flag.PUBSUB, NOSCRIPT, LOADING, STALE)),
-  PUNSUBSCRIBE(new PunsubscribeExecutor(), SUPPORTED,
+  PUNSUBSCRIBE(new PunsubscribeExecutor(), Category.PUBSUB, SUPPORTED,
       new Parameter().min(1).firstKey(0).flags(Flag.PUBSUB, NOSCRIPT, LOADING, STALE)),
-  UNSUBSCRIBE(new UnsubscribeExecutor(), SUPPORTED,
+  UNSUBSCRIBE(new UnsubscribeExecutor(), Category.PUBSUB, SUPPORTED,
       new Parameter().min(1).firstKey(0).flags(Flag.PUBSUB, NOSCRIPT, LOADING, STALE)),
-  PUBSUB(new PubSubExecutor(), SUPPORTED,
+  PUBSUB(new PubSubExecutor(), Category.PUBSUB, SUPPORTED,
       new Parameter().min(2).firstKey(0).flags(Flag.PUBSUB, RANDOM, LOADING, STALE)),
 
-  /************* Cluster *****************/
-  CLUSTER(new ClusterExecutor(), SUPPORTED, new Parameter().min(2)
-      .custom(ClusterParameterRequirements.checkParameters()).firstKey(0)
-      .flags(ADMIN, RANDOM, STALE)),
+  /************* Uncategorized *****************/
+
+  INFO(new InfoExecutor(), Category.UNCATEGORIZED, SUPPORTED,
+      new Parameter().min(1).max(2, ERROR_SYNTAX).firstKey(0)
+          .flags(RANDOM, LOADING, STALE)),
+  LOLWUT(new LolWutExecutor(), Category.UNCATEGORIZED, SUPPORTED,
+      new Parameter().min(1).firstKey(0).flags(READONLY, FAST)),
+
 
   /***************************************
    ******** Unsupported Commands *********
    ***************************************/
 
-  /*************** Connection *************/
-
-  SELECT(new SelectExecutor(), UNSUPPORTED, new Parameter().exact(2).firstKey(0).flags(LOADING,
-      STALE,
-      FAST)),
+  /*************** Keyspace ******************/
 
-  /*************** Keys ******************/
-
-  SCAN(new ScanExecutor(), UNSUPPORTED, new Parameter().min(2).even(ERROR_SYNTAX).firstKey(0).flags(
-      READONLY,
-      RANDOM)),
-  UNLINK(new DelExecutor(), UNSUPPORTED, new Parameter().min(2).lastKey(-1).flags(WRITE, FAST)),
-
-  /************** Strings ****************/
-
-  BITCOUNT(new BitCountExecutor(), UNSUPPORTED, new Parameter().min(2).flags(READONLY)),
-  BITOP(new BitOpExecutor(), UNSUPPORTED,
+  DBSIZE(new DBSizeExecutor(), Category.KEYSPACE, UNSUPPORTED,
+      new Parameter().exact(1).firstKey(0).flags(READONLY, FAST)),
+  FLUSHALL(new FlushAllExecutor(), Category.KEYSPACE, UNSUPPORTED,
+      new Parameter().min(1).max(2, ERROR_SYNTAX).firstKey(0).flags(WRITE)),
+  FLUSHDB(new FlushAllExecutor(), Category.KEYSPACE, UNSUPPORTED,
+      new Parameter().min(1).max(2, ERROR_SYNTAX).firstKey(0).flags(WRITE)),
+  SCAN(new ScanExecutor(), Category.KEYSPACE, UNSUPPORTED,
+      new Parameter().min(2).even(ERROR_SYNTAX).firstKey(0).flags(READONLY, RANDOM)),
+  SELECT(new SelectExecutor(), Category.KEYSPACE, UNSUPPORTED,
+      new Parameter().exact(2).firstKey(0).flags(LOADING, STALE, FAST)),
```

Comment:
I think select still needs to be in the connection section. In the redis docs all the related commands are connection commands

Reply:
The category information is derived from the categories returned by the `INFO` command. So against Redis, `INFO SELECT` lists the categories as `@keyspace` and `@fast`. Using these values allows us to verify against a stricter source (any given redis version). The docs are a lot less specific and could easily re-categorize commands without us knowing.

Reply:
Or were you referring to just where in the file this entry appears?

Reply:
yes, I was just referring to its location. but if the info command has it in that section then that's fine. makes sense to use the info command to categorize our commands